### ISSUE TITLE
When bugbot comments suggesting you request uplift it would be helpful if it told you how.

### ISSUE THREAD

**Issue by Mossop** [mozilla/bugbot#2552]
In [this example comment](https://bugzilla.mozilla.org/show_bug.cgi?id=1933261#c13) bugbot has suggested I request uplift of a patch. But I cannot remember how. It would be helpful if bugbot linked to relevant documentation

**Comment by suhaibmujahid** [comment]
I agree; it would be helpful to link to the docs (https://wiki.mozilla.org/Release_Management/Requesting_an_Uplift).
